### DISCUSSION TITLE
Keywords in example sentences don't have furigana anymore

### DISCUSSION THREAD

**Discussion by unectomy** [Jitendex/Jitendex#97] [Ideas]
I recently updated to the latest version of Jitendex and noticed that the furigana above keywords in example sentences isn't there anymore. What happened to the furigana? Is it a bug?
The furigana was helpful when reading the example sentences because I could keep my eyes on the sentence. Without the furigana above keywords, I have to scroll back up to the searched term to see the it then scroll back to the sentence to keep reading. It's quite inconvenient, especially on longer entries.

**Comment by stephenmk** [comment]
This was a deliberate design decision. I'm aware that it could be inconvenient on longer entries. Unfortunately I think this is the best possible setup that is currently available.

Related discussions: #52 #66

**Reply by unectomy** [reply]
Is there CSS to add the furigana back, or is the furigana gone forever?

**Reply by stephenmk** [reply]
I could add this ability if yomitan supported more embedded CSS styles (namely `display: none`), but that's currently not the case. I see there's been some work on adding support for new CSS stylesheets to yomitan recently, so maybe it will be possible in the near future.

**Comment by stephenmk** [comment]
Latest versions now include the furigana again (see https://github.com/stephenmk/Jitendex/issues/102)
